drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/src/main/java/org/drools/workbench/services/verifier/plugin/client/builders/Utils.java

```diff
@@ -55,4 +74,59 @@ public static DTCellValue52 getRealCellValue(final DTColumnConfig52 config52,
             return visibleCellValue;
         }
     }
+
+    public static Short tryShort(String stringValue) {
+        try {
+            return new Short(stringValue);
+        } catch (final NumberFormatException nfe) {
+            return null;
+        }
+    }
+
+    public static Long tryLong(String stringValue) {
+        try {
+            return new Long(stringValue);
+        } catch (final NumberFormatException nfe) {
+            return null;
+        }
+    }
+
+    public static Double tryDouble(String stringValue) {
+        try {
+            return new Double(stringValue);
+        } catch (final NumberFormatException nfe) {
+            return null;
+        }
+    }
+
+    public static BigInteger tryBigInteger(String stringValue) {
+        try {
+            return new BigInteger(stringValue);
+        } catch (final NumberFormatException nfe) {
+            return null;
+        }
+    }
+
+    public static BigDecimal tryBigDecimal(String stringValue) {
+        try {
+            return new BigDecimal(stringValue);
+        } catch (final NumberFormatException nfe) {
+            return null;
+        }
+    }
+
+    public static Integer tryInteger(String stringValue) {
+        try {
+            return new Integer(stringValue);
+        } catch (final NumberFormatException nfe) {
+            return null;
+        }
+    }
+    public static Boolean tryBoolean(String stringValue) {
+        try {
+            return new Boolean(stringValue);
+        } catch (final NumberFormatException nfe) {
```

Comment:
This is not correct, I checked the Boolean constructor javadoc, no exception is thrown.